### ISSUE TITLE
Import all doesn't work well with ESLint and eslint-plugin-react

### ISSUE THREAD

**Issue by lencioni** [trotzig/import-js#56]
Since we are adding the ESLint control comment to the top of the file for detecting undefined variables

```js
/* eslint no-unused-vars: [2, { "vars": "all", "args": "none" }] */
```

this will work well for regular variables. However, if you are using JSX and React, we need an additional option and access to the eslint-plugin-react plugin, since that plugin is responsible for finding reference errors for components. https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-undef.md

```js
/* eslint react/jsx-no-undef: 2 */
```

**Comment by lencioni** [comment]
I'm actually having a difficult time getting the import all functionality to work for anything with ESLint. I think this is because `eslint` is not executed from the root of the project, so the `.eslintrc` file is not being used. In the project I am trying this in, there are some important options in there that need to be set in order for ESLint to work at all--e.g. ES6 and JSX support.

I wonder instead of adding these comments to the top of the file contents we should require the rules to be specified in the `jshint_cmd` configuration. For ESLint, I think that using a combination of the `--plugin` and `--rule` flags, this could be made to work. Granted, there's nothing stopping us from doing this now, but because this exists it seems like it should just work.

**Comment by lencioni** [comment]
Unfortunately, I don't really see a way to enable JSX support using CLI flags alone. It looks like it is part of the `es6` `env`, but using `--env es6` gives me `<text>: line 82, col 8, Error - Unexpected token <`.

**Comment by trotzig** [comment]
That's a bummer. It's working well for me in the brigade repo, both for regular variables and jsx ones. Is that not the case for you? I'm assuming you're referring to another project above though.

**Comment by lencioni** [comment]
No, I'm referring to the Brigade repo. Do you happen to have a `.eslintrc` file in your home directory or something?

**Comment by trotzig** [comment]
Nope. 

![import_all](https://cloud.githubusercontent.com/assets/578277/11191285/d2243462-8c9a-11e5-8faf-f1b75b2ee75f.gif)

**Comment by trotzig** [comment]
There was a bug with how the eslint comment was injected into the file. Fixed in 3477032977c3516198923a5bce606fc84a89d744. I also made sure that parse errors aren't swallowed in 6cedbecfe7ce1ed4d60d6c7824afd45189831db3.
